pkg/plugin/system/system.go

```diff
@@ -30,10 +30,11 @@ import (
 // PluginSystem is the interface that defines all the methods for the aspect CLI
 // plugin system intended to be used by the Core.
 type PluginSystem interface {
-	Configure(streams ioutils.Streams) error
+	ConfigurePluginSystem(streams ioutils.Streams) error
```

Comment:
Do we need the word "Plugin" or "System" Within the "Plugin System" class?

Reply:
Just one `Configure` method again